syntaxes/v.tmLanguage.json

```diff
@@ -1024,186 +860,167 @@
 				}
 			]
 		},
-		"std-types": {
+		"punctuation": {
 			"patterns": [
 				{
-					"name": "storage.type.boolean.v",
-					"match": "\\bbool\\b"
+					"match": "\\.",
+					"name": "punctuation.definition.other.dot.v"
 				},
 				{
-					"name": "storage.type.byte.v",
-					"match": "\\bbyte\\b"
+					"match": "\\,",
```

Comment:
`\\,` doesn't need escaping. 😉

Additionally, the scope name should ideally be `punctuation.delimiter.comma.v` (see my previous comment).